src/Microsoft.DotNet.Watcher.Tools/Internal/FileSetWatcher.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Tools.Internal;
```

Comment:
Yes. I moved an API usage in this (and the other files) into a different namespace